src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -61,40 +61,124 @@
     {{/unless}} {{asLowerCamelCase name~}}
     {{~#if isArray~}} [] {{~/if}} = {{code}};
   {{/chip_server_cluster_attributes}}
-  {{#chip_cluster_commands}}
-  {{#if arguments}}
+  {{#if (is_server side)}}
+    {{#all_incoming_commands_for_cluster name 'server'}}
```

Comment:
This is a copy an paste with conditions of is_server/all_commands_server and is_client/all_commands_client. Please move these in some js looping so I can read templates like `for all commands in the current cluster`.